cli/tests/integration/run_tests.rs

```diff
@@ -3261,8 +3261,8 @@ itest!(unhandled_rejection_dynamic_import2 {
 });
 
 itest!(nested_error {
-  args: "run run/nested_error.ts",
-  output: "run/nested_error.ts.out",
+  args: "run run/nested_error/main.ts",
+  output: "run/nested_error/main.ts.out",
```

Comment:
I originally moved this file to increase the path length, but I kept it here as part of this PR because it's less cluttered.